src/mumble/CustomElements.cpp

```diff
@@ -147,14 +151,39 @@ void ChatbarTextEdit::setDefaultText(const QString &new_default, bool force) {
 	qsDefaultText = new_default;
 
 	if (bDefaultVisible || force) {
-		QFont f = font();
-		f.setItalic(true);
-		setFont(f);
-		setHtml(qsDefaultText);
-		bDefaultVisible = true;
+		applyPlaceholder();
 	}
 }
 
+void ChatbarTextEdit::applyPlaceholder() {
+	QFont f = font();
+	f.setItalic(true);
+	setFont(f);
+	setWordWrapMode(QTextOption::NoWrap);
+
+	QString placeholderText;
+	bool textModified = false;
+
+	// Minimum chatbar size to trigger elision
```

Comment:
https://github.com/mumble-voip/mumble/blob/6bd2d2e264bd02460177776c838c1ac8de8e89d9/src/mumble/QtWidgetUtils.h#L29

Maybe we can either use the implementation directly or find a way to reuse parts of it so that we don't have text elision implemented independently in different places.

Reply:
Nice, I did not know we already implemented something like this.

Reply:
Yours is much nicer, because it preserves the html formatting :smiling_face_with_tear:

Reply:
It took me ages to get this right though - so it's nice to hear that it can also be used in this context :D